normalize.js

```diff
@@ -111,11 +126,8 @@ define(function() {
     
     return out.substr(0, out.length - 1);
   };
-  
-  var normalizeCSS = function(source, fromBase, toBase) {
 
-    fromBase = removeDoubleSlashes(fromBase);
-    toBase = removeDoubleSlashes(toBase);
+  var normalizeCSS = function(source, fromBase, toBase) {
```

Comment:
ok, I checked ur code locally, and test are fine. All tests using `convertURIBase()` passed.